src/Service/OrderStateImageService.php

```diff
@@ -35,12 +35,33 @@
 
 namespace Mollie\Service;
 
-class ImageService
+class OrderStateImageService
 {
 	public function createOrderStateLogo($orderStateId)
 	{
 		$source = _PS_MODULE_DIR_ . 'mollie/views/img/logo_small.png';
 		$destination = _PS_ORDER_STATE_IMG_DIR_ . $orderStateId . '.gif';
 		@copy($source, $destination);
 	}
+
+	public function deleteOrderStateLogo($orderStateId)
```

Comment:
Annotations for parameters would be nice :shrug:

Reply:
added